lib/action_subscriber.rb

```diff
@@ -57,14 +56,26 @@ def self.configure
     yield(configuration) if block_given?
   end
 
+  def self.draw_routes(&block)
+    routes = Router.draw_routes(&block)
+    @route_set = RouteSet.new(routes)
+  end
+
   def self.print_subscriptions
-    ::ActionSubscriber::Base.print_subscriptions
+    puts configuration.inspect
```

Comment:
@quixoten I am planning on switching all `puts` to `Logger`s when I tackle #25, but right now ActionSubscriber doesn't handle any logging, just standard out